lib/cs_guide/resources/resources.ex

```diff
@@ -5,6 +5,7 @@ defmodule CsGuide.Resources do
 
   import Ecto.Query, warn: false
   alias CsGuide.Repo
+  @ex_aws Application.get_env(:cs_guide, :ex_aws)
```

Comment:
I think the `@ex_aws` constant needs to be defined for the prod and dev configuration. You've done it for test environment by defining it in `config/test.exs` file but I think you forgot to do it for the other configs.
So at the moment I have the error defined here https://github.com/club-soda/club-soda-guide/pull/454#issuecomment-462778871